internal/identity/oidc/gitlab/gitlab.go

```diff
@@ -16,7 +16,7 @@ import (
 // Name identifies the GitLab identity provider.
 const Name = "gitlab"
 
-var defaultScopes = []string{oidc.ScopeOpenID, "profile", "email", "api"}
+var defaultScopes = []string{oidc.ScopeOpenID, "email", "openid"}
```

Comment:
```suggestion
var defaultScopes = []string{oidc.ScopeOpenID, "email", "profile"}
```
oidc.ScopeOpenID is `openid`

Reply:
I still feel like `profile` should be included to be in compliance with the openid spec. Unless it doesn't work?

Reply:
Dropped the commit.